test/Microsoft.TestPlatform.TestUtilities/TempDirectory.cs

```diff
@@ -54,27 +61,28 @@ public void CopyAll(DirectoryInfo source, DirectoryInfo target)
     /// </returns>
     internal static string CreateUniqueDirectory()
     {
-        // AGENT_TEMPDIRECTORY is AzureDevops variable, which is set to path
-        // that is cleaned up after every job. This is preferable to use over
-        // just the normal temp.
         var temp = GetTempPath();
-        var directoryPath = System.IO.Path.Combine(temp, Guid.NewGuid().ToString("n"));
+        var directoryPath = IO.Path.Combine(temp, "vstest", RandomId.Next());
```

Comment:
adding base directory for our output, because we sometimes leave it behind, and it is easier to delete all of them. It is also easier to find the logs of last test that ran if you order by date created.